src/ontology/mondo-edit.obo

```diff
@@ -520167,6 +520168,32 @@ is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditar
 relationship: has_characteristic MONDO:0021152 {source="OMIMPS:616033"} ! inherited
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6777" xsd:anyURI
 
+[Term]
+id: MONDO:0800453
```

Comment:
## Juvenile absence epilepsy changes
I see that all the `mondo-edit.obo` changes are strictly related to these terms. I'm no curator, but LGTM!